frontend/src/utils/search/generateAgencyFilterLookup.ts

```diff
@@ -0,0 +1,21 @@
+import { agencyFilterList } from "src/components/search/SearchFilterAccordion/filterJSONLists/agencyFilterList";
```

Comment:
You do the list in a separate PR? (Is it mostly there? Looks like it's just commented out.)

Reply:
Heh. Never mind. You just pushed more.

Reply:
Pardon me -  I was probably tagging folks to review before the final push